README.md

```diff
@@ -33,6 +33,8 @@ https://user-images.githubusercontent.com/44473782/160729779-41fe207c-b5aa-4fed-
 ## Table of Contents
 
 - [Fixing errors](#Fixing-errors)
+- [Others]
+   -[mov-cli](https://github.com/mov-cli/mov-cli)
```

Comment:
This should include much more of our neighbourhood.
A few to consider:
https://github.com/iamchokerman/ani-sync
https://github.com/justfoolingaround/animdl
https://github.com/saikou-app/saikou
https://github.com/sdaqo/anipy-cli
https://github.com/CoolnsX/dra-cla
https://github.com/Soviena/kaa.si-cli
https://github.com/MangDL/MangDL